From: Etienne Millon <etienne.millon@gmail.com>
Date: Tue, 30 Nov 2010 19:02:39 +0100
Subject: removed-license-in-html-doc

The license in HTML format is redundant with respect to the copyright
information available at /usr/share/doc/zsnes/copyright. This patch prevents the
installation of license.htm and replaces references to it by the former.
---
 docs/readme.htm/about.htm    |    8 ++++----
 docs/readme.htm/advanced.htm |    6 +++---
 docs/readme.htm/faq.htm      |    6 +++---
 docs/readme.htm/games.htm    |    6 +++---
 docs/readme.htm/gui.htm      |    8 ++++----
 docs/readme.htm/history.htm  |    6 +++---
 docs/readme.htm/index.htm    |    8 ++++----
 docs/readme.htm/netplay.htm  |    6 +++---
 docs/readme.htm/readme.htm   |    8 ++++----
 docs/readme.htm/support.htm  |    6 +++---
 10 files changed, 34 insertions(+), 34 deletions(-)

diff --git a/docs/readme.htm/about.htm b/docs/readme.htm/about.htm
index 5d8135b..9b82bf0 100644
--- a/docs/readme.htm/about.htm
+++ b/docs/readme.htm/about.htm
@@ -14,9 +14,9 @@
   <link rel="contents" href="index.htm" title="Index" />
   <link rel="start" href="readme.htm" title="Readme" />
   <link rel="previous" href="history.htm" title="History" />
-  <link rel="next" href="license.htm" title="License" />
+  <link rel="next" href="file:///usr/share/doc/zsnes/copyright" title="License" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -51,7 +51,7 @@
         <li><a href="#credits">Credits</a></li>
       </ul>
     </li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -255,7 +255,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/advanced.htm b/docs/readme.htm/advanced.htm
index 5ad8393..a185f2c 100644
--- a/docs/readme.htm/advanced.htm
+++ b/docs/readme.htm/advanced.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="netplay.htm" title="Netplay" />
   <link rel="next" href="games.htm" title="Games" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -53,7 +53,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -817,7 +817,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/faq.htm b/docs/readme.htm/faq.htm
index 8156eb6..bc7ad87 100644
--- a/docs/readme.htm/faq.htm
+++ b/docs/readme.htm/faq.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="games.htm" title="Games" />
   <link rel="next" href="support.htm" title="Getting Support" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -52,7 +52,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -581,7 +581,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/games.htm b/docs/readme.htm/games.htm
index 2c021c0..f6558b6 100644
--- a/docs/readme.htm/games.htm
+++ b/docs/readme.htm/games.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="advanced.htm" title="Advanced Usage" />
   <link rel="next" href="faq.htm" title="FAQ" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -57,7 +57,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -504,7 +504,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/gui.htm b/docs/readme.htm/gui.htm
index 8630f1e..1e6dc41 100644
--- a/docs/readme.htm/gui.htm
+++ b/docs/readme.htm/gui.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="readme.htm" title="Readme" />
   <link rel="next" href="netplay.htm" title="Netplay" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -56,7 +56,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -1145,7 +1145,7 @@
         <li><strong class="gui">Documentation</strong> button (Win/SDL only) - Opens your default web browser and loads the <a href="http://zsnes-docs.sourceforge.net" title="ZSNES Documentation Home Page">ZSNES Documentation Home Page</a>.</li>
         <li><strong class="gui">Coded By:</strong> - Main developers are listed.  More information in the <a href="about.htm" title="About ZSNES">About</a> page of the documentation.</li>
         <li><strong class="gui">Assistant Coders:</strong> - Assistant developers are listed.  More information in the <a href="about.htm" title="About ZSNES">About</a> page of the documentation.</li>
-        <li>Disclaimer.  Please read the <a href="license.htm" title="GNU General Public License">GNU General Public License</a>.</li>
+        <li>Disclaimer.  Please read the <a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">GNU General Public License</a>.</li>
       </ul>
     </div>
   </div>
@@ -1214,7 +1214,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/history.htm b/docs/readme.htm/history.htm
index 3665ea8..2ae2841 100644
--- a/docs/readme.htm/history.htm
+++ b/docs/readme.htm/history.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="support.htm" title="Getting Support" />
   <link rel="next" href="about.htm" title="About" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -45,7 +45,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -2693,7 +2693,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/index.htm b/docs/readme.htm/index.htm
index 35b58ef..e11b24d 100644
--- a/docs/readme.htm/index.htm
+++ b/docs/readme.htm/index.htm
@@ -16,7 +16,7 @@
 
   <link rel="next" href="readme.htm" title="Readme" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -45,7 +45,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -154,7 +154,7 @@
           <li><a href="about.htm#credits">Credits</a></li>
         </ul>
       </li>
-      <li><a href="license.htm" title="GNU General Public License">License</a></li>
+      <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
     </ul>
   </div>
 
@@ -171,7 +171,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/netplay.htm b/docs/readme.htm/netplay.htm
index fa383de..5f23bd6 100644
--- a/docs/readme.htm/netplay.htm
+++ b/docs/readme.htm/netplay.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="gui.htm" title="GUI" />
   <link rel="next" href="advanced.htm" title="Advanced Usage" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -54,7 +54,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -161,7 +161,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/readme.htm b/docs/readme.htm/readme.htm
index 07c35fe..205387b 100644
--- a/docs/readme.htm/readme.htm
+++ b/docs/readme.htm/readme.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="index.htm" title="Index" />
   <link rel="next" href="gui.htm" title="GUI" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -60,7 +60,7 @@
     <li><a href="support.htm" title="How to get support">Getting Support</a></li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -83,7 +83,7 @@
 
   <div class="section" id="disclaimer">
     <h3>Disclaimer</h3>
-    <p>The ZSNES Development Team, including all developers and contributors, is in no way responsible for any damage caused by the use of this software. Please read the <a href="license.htm" title="GNU General Public License">license</a> for more details.</p>
+    <p>The ZSNES Development Team, including all developers and contributors, is in no way responsible for any damage caused by the use of this software. Please read the <a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">license</a> for more details.</p>
     <p>Due to legal issues, the ZSNES Development Team can provide you neither with <a href="games.htm#roms" title="Description of ROMs">ROMs</a> nor links to them.  In addition, ZSNES <strong>may not be distributed with ROM images.</strong>  However, as with many cases like this, <a href="http://www.google.com" title="Google.com web site">Google</a> is your friend.</p>
     <p>There are still many bugs left in ZSNES, so don't expect it to run all your favorite games.  If ZSNES doesn't work for you, then don't use it.  Use Snes9x, SNEeSe, Super Sleuth, or bsnes instead!  In fact, even if you use ZSNES, use those emulators too!</p>
   </div>
@@ -921,7 +921,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
diff --git a/docs/readme.htm/support.htm b/docs/readme.htm/support.htm
index e0d848b..cd9661d 100644
--- a/docs/readme.htm/support.htm
+++ b/docs/readme.htm/support.htm
@@ -16,7 +16,7 @@
   <link rel="previous" href="faq.htm" title="FAQ" />
   <link rel="next" href="history.htm" title="History" />
   <link rel="help" href="support.htm" title="Getting Support" />
-  <link rel="copyright" href="license.htm" title="GNU General Public License" />
+  <link rel="copyright" href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License" />
 </head>
 
 <body>
@@ -52,7 +52,7 @@
     </li>
     <li><a href="history.htm" title="The log of changes to ZSNES">History</a></li>
     <li><a href="about.htm" title="A brief description of the ZSNES emulator and ZSNES documentation projects">About</a></li>
-    <li><a href="license.htm" title="GNU General Public License">License</a></li>
+    <li><a href="file:///usr/share/doc/zsnes/copyright" title="GNU General Public License">License</a></li>
   </ul>
   <hr />
   <ul>
@@ -187,7 +187,7 @@
     <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Validate this page's CSS"><img style="border:0;width:88px;height:31px" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" /></a>
   </span>
 
-  <p><a href="license.htm" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
+  <p><a href="file:///usr/share/doc/zsnes/copyright" title="View the General Public License">Copyright (C) ZSNES Team &amp; ZSNES Documentation Team</a></p>
 </div>
 
 </div>
